compiler/luci-interpreter/src/kernels/Concatenation.cpp

```diff
@@ -36,20 +36,20 @@ Concatenation::Concatenation(std::vector<const Tensor *> inputs, Tensor *output,
 void Concatenation::configure()
 {
   const int num_inputs = _inputs.size();
-  assert(num_inputs > 0);
+  LUCI_INTERPRETER_CHECK(num_inputs > 0);
```

Comment:
I'm not sure that we need to turn all asserts into conditional throws. This particual condition could be checked earlrier (in luci importer, for example).

Reply:
Umm.... I also agree... but, there is no clear standard for every condition check.